src/auth/auth.service.ts

```diff
@@ -0,0 +1,12 @@
+import { Injectable } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+
+@Injectable()
+export class AuthService {
+  constructor(private configService: ConfigService) {}
+
+  validateApiKey(key: string) {
```

Comment:
We can specify a return type on this function

Reply:
this is updated now